storage-tool.yml

```diff
@@ -415,7 +381,53 @@ services:
           - Paid after that @ $100/TB/Yr. (Consolidated across all paid storage types.)
       - name: max_file_size
         class: 4 TB
+      - name: storage_warmth
+        class: hot
+
+  - name: Hibernate
+    description: |
+      [Hibernate](https://docs.ccv.brown.edu/hibernate/) is a secure, reliable, research data archive solution. Hibernate is a Brown OIT archival service for the research community to migrate inactive data off active Network-attached storage (NAS) platforms onto a lower cost, long-term retention environment. 
+
+        * Hibernate leverages [StarFish](https://docs.ccv.brown.edu/starfish/) an application that provides a metadata and rules-based management framework for large file systems. StarFish makes storage tiering easy: moving data, reporting, zones. 
+        * A zone is a virtual-volume or collection of branches across different file-systems (GPFS, Isilion, LRS etc.). Each PI group is distributed into an individual zone. A zone provides:
+            * Types/sizes of files, access times
+            * Hot Spots (Growth, user(s)/Group(s)
+            * Action: Archive, Delete, Recover
+    features:
+      - name: relative_speed
+        class: slow
+      - name: collaborative_edits
+        class: False
+      - name: shareable_link
+        class: False
+      - name: DOI_provided
+        class: False
+      - name: data_protection
+        class: False
         notes:
+          - up to 6 weeks
```

Comment:
how can it be both false and up to 6 weeks?

Reply:
Good catch! Fixed! Thanks, @mcmcgrath13!!